yamlh.go

```diff
@@ -738,6 +738,7 @@ type yaml_emitter_t struct {
 	explicit_document_start   bool         // Force an explicit document start
 	assume_folded_as_literal  bool         // Assume blocks were scanned as literals
 	indentless_block_sequence bool         // Do not indent block sequences
+	indent_line_comments      int          // The number of indentation spaces before line comments.
```

Comment:
I think another word than "indent" would be good here, just because the term "indent" is overloaded with the indent level in the document. I think the word "pad" might be better here.

Reply:
Quick question, `pad_line_comments` or `pad_line_comment` ?

Reply:
Let's go with singular, `pad_line_comment`

Reply:
Damn I made the wrong choice

Reply:
I'm actually fine with either, so if you haven't got a correction together yet don't worry about it

Reply:
Okay okay, I'll keep `pad_line_comments` then 👍